src/geotools/src/test/java/org/locationtech/geogig/geotools/data/GeoGigDataStoreFactoryTest.java

```diff
@@ -63,24 +60,19 @@ public void testDataStoreFinder() throws Exception {
 
     @Test
     public void testCanProcess() {
-        final File workingDir = repoDirectory;
-
         Map<String, Serializable> params = ImmutableMap.of();
         assertFalse(factory.canProcess(params));
         params = ImmutableMap.of(REPOSITORY.key,
-                (Serializable) (workingDir.getName() + "/testCanProcess"));
+                (Serializable) (repositoryDirectory.getName() + "/testCanProcess"));
         assertFalse(factory.canProcess(params));
 
-        params = ImmutableMap.of(REPOSITORY.key, (Serializable) workingDir.getAbsolutePath());
-        assertTrue(factory.canProcess(params));
```

Comment:
this ends up being the same test assertion as
```
params = ImmutableMap.of(REPOSITORY.key, (Serializable) repoDirectory.getAbsolutePath());
```
on line 80 below (or 72 in my changed version), since **workingDir** is the same as the parent **repositoryDirectory**.